docs/csharp/programming-guide/classes-and-structs/named-and-optional-arguments.md

```diff
@@ -50,28 +50,36 @@ translation.priority.ht:
  Named and optional parameters, when used together, enable you to supply arguments for only a few parameters from a list of optional parameters. This capability greatly facilitates calls to COM interfaces such as the Microsoft Office Automation APIs.  
   
 ## Named Arguments  
- Named arguments free you from the need to remember or to look up the order of parameters in the parameter lists of called methods. The parameter for each argument can be specified by parameter name. For example, a function that calculates body mass index (BMI) can be called in the standard way by sending arguments for weight and height by position, in the order defined by the function.  
+ Named arguments free you from the need to remember or to look up the order of parameters in the parameter lists of called methods. The parameter for each argument can be specified by parameter name. For example, a function that prints order details (such as, seller name, order number & product name) can be called in the standard way by sending arguments by position, in the order defined by the function.
   
- `CalculateBMI(123, 64);`  
+ `PrintOrderDetails("Gift Shop", 31, "Red Mug");`
   
- If you do not remember the order of the parameters but you do know their names, you can send the arguments in either order, weight first or height first.  
+ If you do not remember the order of the parameters but know their names, you can send the arguments in any order.  
   
- `CalculateBMI(weight: 123, height: 64);`  
+ `PrintOrderDetails(orderNum: 31, productName: "Red Mug", sellerName: "Gift Shop");`
   
- `CalculateBMI(height: 64, weight: 123);`  
+ `PrintOrderDetails(productName: "Red Mug", sellerName: "Gift Shop", orderNum: 31);`
   
  Named arguments also improve the readability of your code by identifying what each argument represents.  
   
- A named argument can follow positional arguments, as shown here.  
-  
- `CalculateBMI(123, height: 64);`  
-  
- However, a positional argument cannot follow a named argument. The following statement causes a compiler error.  
+ Named arguments, when used with positional arguments, are valid as long as 
+
+- they're not followed by any positional arguments, or
+
+ `PrintOrderDetails("Gift Shop", 31, productName: "Red Mug");`
+
+- _starting with C# 7.2_, they're used in the correct position.
```

Comment:
I would add a a note here that the 2nd argument is not named. Just to call it out explicitly and make it really clear what you are demonstrating.